src/ops/controlflow.rs

```diff
@@ -166,6 +170,15 @@ impl OpTrait for BasicBlock {
     fn other_output(&self) -> Option<EdgeKind> {
         Some(EdgeKind::ControlFlow)
     }
+
+    fn signature(&self) -> FunctionType {
+        match self {
+            BasicBlock::DFB {
+                extension_delta, ..
+            } => FunctionType::new(type_row![], type_row![]).with_extension_delta(extension_delta),
```

Comment:
This is a hack to make extension inference work! Previously the signature was just the empty default